src/components/MyrTour.js

```diff
@@ -39,7 +49,11 @@ const steps = [
   {
     selector: '#ace-editor',
     content: 'This is the editor. You can create 3D scenes using JavaScript ' +
-      'and a special set of instructions or functions to MYR.',
+      'and a special set of instructions or functions to MYR.'
+  },
+  {
+    selector: '#configure-scene',
```

Comment:
This could end up a little confusing where we go to the scene config menu button twice. Let's just add this note within the editor tour itself.

Reply:
fair enough. will do.